### PR TITLE
fix: heredoc/nowdoc trailing newlines

### LINKED ISSUE #1292: Nowdoc: invalid trailing comma
```php
call(
    $var
        ? "foo"
        : <<<'EOD'
        string
        string
        string
        EOD
);
```
is formatted as
```php
call(
    $var
        ? "foo"
        : <<<'EOD'
        string
        string
        string
        EOD,
);
```
which is invalid.

### PR DESCRIPTION
**Pull request by czosel**
Fixes #1292

### CODE REVIEW

**Comment by czosel** [comment]
@evilebottnawi can you take a look at thise one as well? I have the release notes for 0.14.0 already prepared :wink:

**Comment by czosel** [comment]
@evilebottnawi this is the last one :wink:

File: tests/nowdoc/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -929,8 +929,7 @@ string
 string
 string
 EOD
-    ,
-    $var
+    , $var
```
Comment (evilebottnawi) [inline tests/nowdoc/__snapshots__/jsfmt.spec.js.snap]: Why it is changed?

Comment (czosel) [reply tests/nowdoc/__snapshots__/jsfmt.spec.js.snap]: I think this is more of a side effect, the main idea of the PR was to fix #1292. It doesn't look too bad though, does it?

Comment (czosel) [reply tests/nowdoc/__snapshots__/jsfmt.spec.js.snap]: @evilebottnawi what do you say, good to go? This is now literally the last open issue for our release :wink:

Comment (evilebottnawi) [reply tests/nowdoc/__snapshots__/jsfmt.spec.js.snap]: I think will be better to put it on new line, because we always put each argument on next line

Comment (czosel) [reply tests/nowdoc/__snapshots__/jsfmt.spec.js.snap]: Think we can do that in a separate PR?

Comment (evilebottnawi) [reply tests/nowdoc/__snapshots__/jsfmt.spec.js.snap]: Yes, just open a new issue

Comment (czosel) [reply tests/nowdoc/__snapshots__/jsfmt.spec.js.snap]: done! I'll make the release now :tada:

**Pull request merged**